### DISCUSSION TITLE
vitepress如何设置中文

### DISCUSSION THREAD

**Discussion by zeMinng** [vuejs/vitepress#3302] [Q&A] [answered]
"vitepress": "^1.0.0-rc.31"版本
如图，请问如何设置成中文？

尝试在config.js设置但不生效，请求帮助
locales: {
    '/': {
      lang: 'zh-CN',
    }
  }
![20231207-171448](https://github.com/vuejs/vitepress/assets/75962652/96775d60-f55b-4bf5-9131-7b3051be881b)

**Comment by zeMinng** [comment]
"vitepress": "^1.0.0-rc.31" version
As shown in the figure, how to set it up in Chinese?



 Try to set up in config.js but not effective, request help
locales: {
'/': {
lang: 'zh-CN',
}
}

**Comment by Zhengqbbb** [comment]
参考： 
- https://github.com/vitejs/docs-cn/blob/e81a0cba9275cef218774ec936fddb4b0b3322e9/.vitepress/config.ts#L99-L164
- 如图中 `Next page` 都可以自行通过 `themeConfig.docFooter ` 进行修改：https://vitepress.dev/reference/default-theme-config#docfooter

**Reply by zeMinng** [reply]
为什么defineConfig()方式不可以，module.exports可以，help
Why can't defineConfig() way, module.exports can, help

![image](https://github.com/vuejs/vitepress/assets/75962652/bbea75c5-0908-4505-a240-d64203c76ea2)

**Reply by brc-dd** [reply]
@2439340964 You need to name your file config.mjs or config.mts. If you want to keep using config.js or config.ts, you need to add `"type": "module"` in your package.json. The link in that error message clearly indicates how to fix that.

**Comment by zeMinng** [comment]
根据文档设置的lang: 'zh-CN',没有什么卵用
According to the documentation set of lang: 'zh-CN', no eggs are used

![image](https://github.com/vuejs/vitepress/assets/75962652/00db6224-cde4-4de8-874d-ac0a691a195d)
![image](https://github.com/vuejs/vitepress/assets/75962652/364a19bc-a035-42bc-9ba5-d0eeac5956e3)

**Reply by maicss** [reply]
问别人问题使用脏话，而且直译之后脏话更没有任何可以理解的可能，也不检查，直接放上来，现实中久这样请教别人的？

**Reply by zeMinng** [reply]
请问您？我哪有使用脏话或者攻击性言语？您觉得我说话带有攻击性、不好的言论，我想您此刻最需要的是举报我。

**Reply by maicss** [reply]
卵是什么，no eggs are used 是什么？

**Reply by zeMinng** [reply]
我使用的是汉译英翻译过来可能是无效的一段话，倒不至于含有攻击性言语。如果您觉得'卵'这个词含有攻击性、辱骂，想必您使用vpn来访问github/internet也不具有任何任意。
至于我的态度，也不是您使用'卵'一词就可以定性谢谢，请您首尾呼应，全文贯穿的看。

**Reply by maicss** [reply]
为什么别人怎么上网跟你说的什么话有关系？这样能反证你素质？不再回复了。

**Reply by zeMinng** [reply]
您不妨去违禁词百科里找找，我这句话构成辱骂、攻击性？

**Reply by maicss** [reply]
你不觉得啊，那没事了。

**Reply by awxiaoxian2020** [reply]
你是说什么地方需要设置成中文？

**Reply by zeMinng** [reply]
谢谢，已经成功解决过了，可以关闭我这个讨论。
在config.ts中是可配置的，之后我尝试进行了配置是成功的

**Comment by kaikaibenkai** [comment]
`lang: 'zh-CN'` means `<html lang="zh-CN">`

**Comment by s-theo** [comment, marked as answer]
```ts
themeConfig: {   
 
    // 文章翻页
    docFooter: {
      prev: '上一篇',
      next: '下一篇'
    },

    // 移动端 - 外观
    darkModeSwitchLabel: '外观',

    // 移动端 - 返回顶部
    returnToTopLabel: '返回顶部',

    // 移动端 - menu
    sidebarMenuLabel: '菜单',
}
```

<img width="344" alt="image" src="https://github.com/vuejs/vitepress/assets/89436345/1022ee54-962f-452b-9769-3042d2292da3">

**Reply by zeMinng** [reply]
是的，这是个正确的配置

**Comment by Alaye-Dong** [comment]
这有一个插件 [vitepress-i18n](https://github.com/jooy2/vitepress-i18n) ，可以翻译默认主题的文本，效果不错，你可以试试
